src/ucp/tag/rndv.c

```diff
@@ -422,8 +422,6 @@ void ucp_tag_rndv_cancel(ucp_request_t *sreq)
     if (!(sreq->send.ep->flags & UCP_EP_FLAG_REMOTE_CONNECTED)) {
         if (sreq->flags & UCP_REQUEST_FLAG_RNDV_RTS_SENT) {
             ucp_rndv_req_add_to_cancelled_list(sreq, UCS_ERR_CANCELED);
-        } else {
```

Comment:
when is this operation completed if rts is not sent?
is it still on pending queue in this case?

Reply:
maybe add code comment

Reply:
> when is this operation completed if rts is not sent?

it will be completed during pending purge

> is it still on pending queue in this case?

yes, it is still in the pending queue, and it will be removed when purging

Reply:
> maybe add code comment

done